py/client/pydeephaven/table.py

```diff
@@ -855,7 +855,8 @@ def multi_join(input: Union[Table, Sequence[Table], MultiJoinInput, Sequence[Mul
             MultiJoinInput objects are supplied, this parameter must be omitted.
 
     Returns:
-        MultiJoinTable: the result of the multi-table natural join operation.
+        MultiJoinTable: the result of the multi-table natural join operation.  To access the underlying Table, use the
+            :meth:`~MultiJoinTable.table` method.
```

Comment:
This same change is needed in the server docs https://github.com/deephaven/deephaven-core/blob/main/py/server/deephaven/table.py#L3869